client/pages/picker-page.jsx

```diff
@@ -37,15 +37,17 @@ export default class PickerPage extends React.Component {
   }
 
   addOutfit() {
-    const userId = this.context.user.userId;
+    const user = this.context.user;
     const token = this.context.token;
-    fetch(`/api/outfits/${userId}`, {
+    const reqBody = this.state;
+    reqBody.user = user;
```

Comment:
This shouldn't be necessary as your server can figure out the user info from the token/database